ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml

```diff
@@ -340,38 +294,27 @@ tests:
     cluster_profile: azure-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
-      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.13 and @azure-upi and not @fips'
-      UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.13 and @azure-upi and not @fips'
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-upi
 - as: gcp-ipi-disc-priv-oidc-f28
   cron: 57 21 23 * *
   steps:
     cluster_profile: gcp-qe
-    env:
```

Comment:
Shall we add `@disconnected` tag here?

Reply:
Yes, we should. 
This PR is cleanup tags that have been optimized and it's already large enough. I think add missed `@disconnected` tag is another task that might be better in a follow up PR.

Reply:
Address the issue in https://github.com/openshift/release/pull/49468